qiskit-qir/tests/test_circuits/random.py

```diff
@@ -0,0 +1,25 @@
+##
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+##
+import pytest
+
+from qiskit import transpile
+from qiskit.circuit.random import random_circuit
+from qiskit_qir.visitor import SUPPORTED_INSTRUCTIONS
+
+
+def generate_random_fixture(num_qubits, depth):
+    @pytest.fixture()
+    def random():
+        circuit = random_circuit(num_qubits, depth, measure=True)
+        return transpile(circuit, basis_gates = SUPPORTED_INSTRUCTIONS)   
```

Comment:
@guenp @cgranade  if we remove `id` from supported instructions, will transpile automatically do `x()->x()`